src/core_plugins/kibana/public/visualize/editor/styles/_editor.less

```diff
@@ -46,8 +46,14 @@
   vis-editor-agg-group {
     .flex-parent(0, 1, auto);
   }
+
+
 }
 
+  .vis-option-item {
```

Comment:
This applies to a number of visualizations and IMO looks rather awkward:

<img width="356" alt="screen shot 2017-03-24 at 8 47 11 am" src="https://cloud.githubusercontent.com/assets/627123/24294827/883c7270-106e-11e7-9be3-0d7f7ddf038f.png">
<img width="356" alt="screen shot 2017-03-24 at 8 47 23 am" src="https://cloud.githubusercontent.com/assets/627123/24294831/89db838c-106e-11e7-8b74-356ae9d5cceb.png">